src/module.c

```diff
@@ -13091,7 +13094,7 @@ NULL
 
 /* Return the number of registered modules. */
 size_t moduleCount(void) {
-    return dictSize(modules);
+    return modules_count;
```

Comment:
let's add a comment as to why we need this variable and we're not using dictSize directly

Reply:
done